addons/l10n_ke_edi_tremol/models/product.py

```diff
@@ -14,6 +14,12 @@ class ProductTemplate(models.Model):
         string='KRA Item Description',
         help='Product code description needed when not 16% VAT rated. ',
     )
+    current_country_code = fields.Char(compute='_compute_current_country_code')
```

Comment:
We already have a field fiscal_country_codes

Reply:
it was added until 16.1 it's not available in 16.0

Reply:
@moerradi Then let's only do it from saas-16.1

Reply:
aright !